ingestion/functions/template.yaml

```diff
@@ -0,0 +1,22 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: AWS Lambda functions for epid ingestion
+
+Resources:
+  RetrievalFunction:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      CodeUri: retrieval/
+      Handler: retrieval.lambda_handler
+      Runtime: python3.8
```

Comment:
Did you need anything that was not available in 3.6 ?
Just asking because the rest of what I've seen in healthmap (frontend, sheets jobs) is based on 3.6, perhaps it will be simpler for them if we stick to it.

Reply:
I don't -- happy to change, thanks for testing.